workers/unity/Packages/io.improbable.gdk.playerlifecycle/Systems/PlayerCreation/SendCreatePlayerRequestSystem.cs

```diff
@@ -65,11 +65,16 @@ private void SendPlayerCreatorEntityQuery()
 
         private void HandleEntityQueryResponses()
         {
+            if (!playerCreatorEntityQueryId.HasValue)
```

Comment:
What does it not having a value mean has happened? How has that changed with the rest of the code, or is this something you found because of the change?

Reply:
Previously a check was done `if(long? == long)` which worked because there is an implicit conversion there. This implicit conversion is gone.

If there is no Value in that nullable nothing is supposed to happen, so I changed it to an early out.